diff --git a/toolchain/build-toolchain.sh b/toolchain/build-toolchain.sh
index 92e6a17..25a9aa5 100755
--- a/toolchain/build-toolchain.sh
+++ b/toolchain/build-toolchain.sh
@@ -12,7 +12,7 @@ log() {
 
 # targets for libphoenix and phoenix-rtos-kernel for installing headers
 declare -A TOOLCHAN_TO_PHOENIX_TARGETS=(
-    [arm-phoenix]="armv7a9-zynq7000 armv7a7-imx6ull armv7m7-imxrt106x armv7m4-stm32l4x6"
+    [arm-phoenix]="armv7a9-zynq7000 armv7a7-imx6ull armv7m7-imxrt106x armv7m4-stm32l4x6 armv8m55-stm32n6"
     [i386-pc-phoenix]="ia32-generic"
     [riscv64-phoenix]="riscv64-generic"
     [sparc-phoenix]="sparcv8leon-gr716 sparcv8leon-gr712rc sparcv8leon-gr740 sparcv8leon-generic"
